src/convertSourceToRaw.m

```diff
@@ -2,33 +2,24 @@
 
 function convertSourceToRaw(cfg)
     %
-    % Short description of what the function does goes here.
-    %
-    % USAGE::
+    % Function attempts to convert a source dataset created with CPP_BIDS into a valid
+    % BIDS data set.
     %
-    %   [argout1, argout2] = templateFunction(argin1, [argin2 == default,] [argin3])
+    % Only covers func folder at the moment
```

Comment:
```suggestion
```

This is not true anymore. I am pretty sure it converts any folder withing a subject folder.